docs/detections/rules-ui-monitor.asciidoc

```diff
@@ -32,18 +31,28 @@ TIP: To sort the rules list, click any column header. To sort in descending orde
 
 For detailed information on a rule, the alerts it generated, and associated errors, click on its name in the table. This also allows you to perform the same actions that are available on the <<rules-ui-management, **Installed Rules** tab>>, such as modifying or deleting rules, activating or deactivating rules, exporting or importing rules, and duplicating prebuilt rules.
 
+For information about rule execution gaps (which are periods of time when a rule didn't run as expected), use the panel above the table. The time filter on the left allows you to select a time range for viewing gap data. The **Total rules with gaps:** field tells you how many rules have unfilled or partially filled gaps within the selected time range. The **Only rules with gaps** filter on the right lets you only display rules with unfilled or partially filled gaps. 
+
+Within the table, the **Last Gap (if any)** column conveys how long the most recent gap for a rule lasted. The **Unfilled gaps duration** column shows whether a rule still has gaps and provides a total sum of the remaining unfilled or partially filled gaps. If a rule has no gaps, these columns display a dash (`––`).
```

Comment:
Unfilled gaps duration depends on the date picker on the Gaps panel, not sure how to phrase to include this information here

Reply:
Ah yes, that's important to call out as well. How about this:

```suggestion
Within the table, the **Last Gap (if any)** column conveys how long the most recent gap for a rule lasted. The **Unfilled gaps duration** column shows whether a rule still has gaps and provides a total sum of the remaining unfilled or partially filled gaps. The total sum can change based on the time range that you select in the panel above the table. If a rule has no gaps, both columns display a dash (`––`).
```

Reply:
Looks good, thank you!